tests/pages/DataOverlayPageObject.ts

```diff
@@ -0,0 +1,33 @@
+import { Page } from '@playwright/test';
+
+export class DataOverlayPageObject {
+  readonly page: Page;
+
+  constructor(page: Page) {
+    this.page = page;
+  }
+
+  get button() {
+    return this.page.locator('[data-cy^="dataOverlayMenu"]').first();
+  }
+
+  async toggle(viewportId: string = 'default') {
+    await this.page.getByTestId(`dataOverlayMenu-${viewportId}-btn`).click();
+  }
+
+  async addSegmentation(segmentation: string, viewportId: string = 'default') {
+    await this.page.getByTestId(`AddSegmentationDataOverlay-${viewportId}`).click();
+    await this.page.getByText('SELECT A SEGMENTATION').click();
+    await this.page.getByTestId(segmentation).click();
+  }
+
+  async changeSegmentation(from: string, to: string) {
+    await this.page.getByTestId(`overlay-ds-select-value-${from.toUpperCase()}`).click();
+    await this.page.getByTestId(`${to}-SEG`).click();
+  }
+
+  async remove() {
```

Comment:
This currently only ever removes the segmentation named (literally) SEGMENTATION. The name of the segmentation overlay should be passed in and templated into the selectors/locators.